migrations/20200721091000-create-hash-column.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('ConnectedAccounts', 'hash', { type: Sequelize.STRING });
```

Comment:
Maybe we could enforce the uniqueness at the DB level for more safety? (except if we have duplicate tokens in the DB today)

```suggestion
    await queryInterface.addColumn('ConnectedAccounts', 'hash', { type: Sequelize.STRING, unique: true });
```

Reply:
I actually coded that way, the problem is that we still have duplicated tokens in our DB that'll require a manual process to solve.
I'll put that back in my priorities list.